js/acquia_lift.debugger.js

```diff
@@ -0,0 +1,1885 @@
+                        }
+                    }
+                }
+                if(returnValue){
+                    returnValue = false;
+                    for (var i = 0; i < $scope.type.length; i++) {
+                        if (typeof($scope.type[i].name) === "string") {
+                            if ($scope.type[i].name.toUpperCase() === row.type.toUpperCase()) {
+                                returnValue =  true;
+                            }
+                        }
+                    }       
+                }
+            }
+        }
+        return returnValue;
+    };
+
+    /**
+     * Adds an item to the list of segments to preview. 
+     * Removes that item from the list in autocomplete box.
+     */
+    $scope.addPreviewItem = function (item) {
+        var value;
+        if (item==null){
+            value = $scope.overrideSegmentChoice;
+        }else{
+            value = item;
+        }
+        var indexAll = $scope.allSegments.indexOf(value);
+        if(indexAll > -1){
+            $scope.allSegments.splice(indexAll, 1);
+            $scope.profile.overrideSegments.push(value);
+        }
+    }
+
+    /**
+     * Removes an item to the list of segments to preview. 
+     * Adds that item from the list in autocomplete box.
+     */
+    $scope.deletePreviewItem = function(item){
+        var index = $scope.profile.overrideSegments.indexOf(item);
+        if(index > -1){
+            $scope.profile.overrideSegments.splice(index, 1);
+        }
+        $scope.allSegments.push(item);
+    }
+
+    /**
+     * Sets override segments for preview
+     * Tells user to refresh page page to see changes
+     */
+    $scope.startPreview = function(){
+        Drupal.acquiaLiftProfilesDebug.setOverrideSegments($scope.profile.overrideSegments);    
+        document.getElementsByClassName('debugger__preview__notification')[0].innerHTML = 'Please refresh page to see your changes. You may have to clear your cache or refresh multiple times.';
+    }
+
+    /**
+     * Changes override segments back to previous versions
+     * clears saved override segments.
+     */
+    $scope.stopPreview = function(){
+        $scope.profile.overrideSegments = $sessionStorage.getObject(debugPrefix + "::overrideSegments");
+        $scope.profile.curSegments = $sessionStorage.getObject(debugPrefix + "::originalSegments");
+        if ($scope.profile.overrideSegments){
+            Drupal.acquiaLiftProfilesDebug.setOverrideSegments(null);
+            $sessionStorage.removeItem(debugPrefix + "::overrideSegments");
+        }
+    }
+
+    /**
+     * Sets button CSS depending on preview status.
+     * Sets Site Preview Tab label.
+     * Sets messages in the Lift Profile tab.
+     */
+    $scope.isPreview = function(){
+        if (window.sessionStorage.getItem("acquiaLift::debug::overrideSegments")){
+            $scope.previewButtonStop = {
+                'background-color': ' #0073b9',
```

Comment:
Can we do this with adding/removing classes?  It would take a long time to hunt down a hex value stored deep in the JavaScript code.